spring-kafka/src/main/java/org/springframework/kafka/listener/ConcurrentMessageListenerContainer.java

```diff
@@ -392,6 +402,8 @@ && getContainerProperties().isRestartAfterAuthExceptions()
 				exec = new SimpleAsyncTaskExecutor(getListenerId() + ".authRestart");
 			}
 			exec.execute(this::start);
+		} else if (this.concurrency == stoppedContainersCount) {
```

Comment:
According to code style the `else` supposed to be on a new line.
Please, run `gradlew check` before pushing to the PR.